src/test/ui/suggestions/issue-84973-blacklist.stderr

```diff
@@ -31,10 +31,12 @@ LL | #[derive(Clone)]
    |
 
 error[E0277]: `[static generator@$DIR/issue-84973-blacklist.rs:17:13: 17:22]` cannot be unpinned
-  --> $DIR/issue-84973-blacklist.rs:17:5
+  --> $DIR/issue-84973-blacklist.rs:17:13
    |
 LL |     f_unpin(static || { yield; });
-   |     ^^^^^^^ the trait `Unpin` is not implemented for `[static generator@$DIR/issue-84973-blacklist.rs:17:13: 17:22]`
+   |     ------- ^^^^^^^^^^^^^^^^^^^^ the trait `Unpin` is not implemented for `[static generator@$DIR/issue-84973-blacklist.rs:17:13: 17:22]`
```

Comment:
I just realized that in cases where the closure would be the only label, and it's pointing at the closure itself, we can customize the output and refer to "this closure" instead.

Reply:
This seems a bit difficult since I'm walking over the unsubstituted predicate here. I'm gonna leave that as a follow-up, since it's not technically a regression I think.

Reply:
Yeah, yeah, this was more of a general observation, not part of the review.